hudi-client/hudi-client-common/src/main/java/org/apache/hudi/execution/CopyOnWriteInsertHandler.java

```diff
@@ -69,9 +73,16 @@ public CopyOnWriteInsertHandler(HoodieWriteConfig config, String instantTime,
 
   @Override
   public void consumeOneRecord(HoodieInsertValueGenResult<HoodieRecord> payload) {
-    final HoodieRecord insertPayload = payload.record;
+    final HoodieRecord<T> insertPayload = payload.record;
     String partitionPath = insertPayload.getPartitionPath();
     HoodieWriteHandle<?,?,?,?> handle = handles.get(partitionPath);
+
+    Option<IndexedRecord> insertRecord = payload.insertValue;
+    // just skip the ignored record，do not make partitions on fs
+    if (insertRecord.isPresent() && insertRecord.get().equals(IGNORE_RECORD)) {
```

Comment:
may be only need to jude when the wirte handle be created

Reply:
@KnightChess  The partition path is created in the writeHandleFactory.create method. If the handle was created, an empty file has been created

Reply:
@weimingdiit after handle judge, before `weriteHandleFactory.create`

Reply:
@KnightChess  ok，get